### PR TITLE
[1LP][RFR] Widgetastic conversion of Chargeback Assignments Form

### PR DESCRIPTION
**Pull request by nachandr**
{{pytest: -v cfme/tests/intelligence/test_chargeback_assignments.py}}

### CODE REVIEW

**Comment by dajoRH** [comment]
Lint report for commit 3a6b22b46c78c1e5c8b102d8ed7cd927bdf75bd0:

`cfme/intelligence/chargeback/assignments.py`:
- :grey_exclamation: Line 92:9: **T001** *print statement found.*

Please, rectify these issues :smirk: .


**No commit flaws detected**.

*CFME QE Bot*

**Comment by dajoRH** [comment]
Lint report for commit 410a05ce226e1bb4f1b213fd3cd278cab8470ae2:

`cfme/intelligence/chargeback/assignments.py`:
- :grey_exclamation: Line 92:9: **T001** *print statement found.*

Please, rectify these issues :smirk: .


**No commit flaws detected**.

*CFME QE Bot*

**Comment by dajoRH** [comment]
Lint report for commit 39c8253590f4d36c6fb0af9fae1e1ec5f607ec4c:

`cfme/intelligence/chargeback/assignments.py`:
- :grey_exclamation: Line 92:9: **T001** *print statement found.*

`cfme/tests/intelligence/test_chargeback_assignments.py`:
- :red_circle: Line 11:1: **F401** *'random.choice' imported but unused*

Please, rectify these issues :smirk: .


**No commit flaws detected**.

*CFME QE Bot*

File: cfme/tests/intelligence/test_chargeback_assignments.py [hunk 1]
```diff
@@ -17,109 +17,130 @@
...
-    selected_option = sel.text(
-        cb.assign_form.selections.select_by_name("Enterprise").first_selected_option)
+    assign_view = view.browser.create_view(AssignmentsView)
+    row = assign_view.selections.row(name='Enterprise')
+    selected_option = row.rate.widget.selected_option
     assert selected_option == "Default", 'Selection does not match'
 
 
 def test_assign_compute_provider(virtualcenter_provider):
+    app = get_or_create_current_appliance()
```
Comment (quarckster) [inline cfme/tests/intelligence/test_chargeback_assignments.py]: **REQUIRED**
I propose to not get the appliance object, but import Server class:
```python
from cfme.base import Server
...
view = navigate_to(Server, 'Chargeback')
```

File: cfme/tests/intelligence/test_chargeback_assignments.py [hunk 2]
```diff
@@ -17,109 +17,130 @@
...
-    selected_option = sel.text(
-        cb.assign_form.selections.select_by_name(virtualcenter_provider.name).first_selected_option)
+    assign_view = view.browser.create_view(AssignmentsView)
+    row = assign_view.selections.row(name=virtualcenter_provider.name)
+    selected_option = row.rate.widget.selected_option
     assert selected_option == "Default", 'Selection does not match'
 
 
 def test_assign_compute_cluster(virtualcenter_provider):
+    app = get_or_create_current_appliance()
```
Comment (quarckster) [inline cfme/tests/intelligence/test_chargeback_assignments.py]: Here as well

File: cfme/tests/intelligence/test_chargeback_assignments.py [hunk 3]
```diff
@@ -17,109 +17,130 @@
...
-    selected_option = sel.text(
-        cb.assign_form.selections.select_by_name(cluster_name).first_selected_option)
+    assign_view = view.browser.create_view(AssignmentsView)
+    row = assign_view.selections.row(name=cluster_name)
+    selected_option = row.rate.widget.selected_option
     assert selected_option == "Default", 'Selection does not match'
 
 
 def test_assign_compute_taggedvm(virtualcenter_provider):
+    app = get_or_create_current_appliance()
```
Comment (quarckster) [inline cfme/tests/intelligence/test_chargeback_assignments.py]: replace by `Server`

File: cfme/tests/intelligence/test_chargeback_assignments.py [hunk 4]
```diff
@@ -17,109 +17,130 @@
...
-        cb.assign_form.selections.select_by_name("Chicago").first_selected_option)
+    assign_view = view.browser.create_view(AssignmentsView)
+    row = assign_view.selections.row(name='Chicago')
+    selected_option = row.rate.widget.selected_option
     assert selected_option == "Default", 'Selection does not match'
 
 
 @pytest.mark.meta(blockers=[1273654])
 def test_assign_storage_enterprise(virtualcenter_provider):
+    app = get_or_create_current_appliance()
```
Comment (quarckster) [inline cfme/tests/intelligence/test_chargeback_assignments.py]: replace by Server

File: cfme/tests/intelligence/test_chargeback_assignments.py (hunk 1 above)
Comment (nachandr) [inline cfme/tests/intelligence/test_chargeback_assignments.py]: @quarckster , Done

**Pull request merged**